sanic/response.py

```diff
@@ -130,12 +130,7 @@ def get_headers(
         )
 
         headers = self._parse_headers()
-
-        if self.status == 200:
-            status = b"OK"
-        else:
-            status = STATUS_CODES.get(self.status)
-
+        status = STATUS_CODES.get(self.status)
         return (b"HTTP/%b %d %b\r\n" b"%b" b"%b\r\n") % (
             version.encode(),
             self.status,
```

Comment:
Please note that unlike line 187, there is no default `UNKNOWN RESPONSE` response here, which means this could be `None` and cause issues. Should this be fixed to mimic the behavior below?